src/Stryker.Core/Stryker.Core/Compiling/FsharpCompilingProcess.cs

```diff
@@ -63,7 +63,6 @@ public CompilingProcessResult Compile(IEnumerable<ParsedInput> syntaxTrees, bool
                     _fileSystem.Directory.CreateDirectory(injectionPath);
                 }
 
-                _fileSystem.File.Copy(mutatedAssemblyPath, injectionPath, true);
```

Comment:
This was dramatically crashing with an IOException - "the project cannot access the file". Which probably makes sense, since at least for the `mutatedAssemblyPath` was the same as `injectionPath`. 

Regardless, just removing this line works :D That said, this should be revisited in the future to align the whole thing with the current C# flow.